### PR TITLE
Breaking changes to config

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-views/issues/34

### CODE REVIEW

**Comment by odscjames** [comment]
Do NOT merge till master til deploy changes done or you'll just break the live deploy!

**Comment by odscjames** [comment]
This comes with https://github.com/open-contracting/kingfisher/pull/328/files

Problem; one of the reasons this was done was so we can remove the setup.py from process. But ...... look in .travis.yml - it needs to install process to make the tests work! But we can maybe merge this, then look at tests.

**Comment by jpmckinney** [comment]
I figure the tests should git clone kingfisher-process and pip install its requirements – like when deploying the process app.

**Comment by odscjames** [comment]
https://github.com/open-contracting/deploy/pull/37 is WIP for deploy repo but it hasn't been tested at all

**Comment by odscjames** [comment]
> I figure the tests should git clone kingfisher-process and pip install its requirements – like when deploying the process app.

I tried this in a separate branch but there are problems. Process is written with a different line length flake8 setting - longer than 119.  So I got the tests to work, but then flake8 failed. To avoid this pull request growing, I'll suggest we keep to this scope and then look into this later in a new issue.

**Comment by jpmckinney** [comment]
> I tried this in a separate branch but there are problems. Process is written with a different line length flake8 setting - longer than 119. So I got the tests to work, but then flake8 failed. To avoid this pull request growing, I'll suggest we keep to this scope and then look into this later in a new issue.

Sounds good. I think easiest might be to fix line lengths in KF Process, as I'm not sure why one project needs a longer line length.

**Comment by jpmckinney** [comment]
I opened https://github.com/open-contracting/kingfisher-views/issues/41

File: requirements.in [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
--e git+https://github.com/open-contracting/kingfisher-process.git@master#egg=ocdskingfisher
-# See libcove, required by kingfisher-process
-jsonschema<2.7
-
 alembic
 logzero
 pgpasslib
 psycopg2
+# Lock to 1.2 branch (1.3 branch has issues with an identifier being too long - this caused problems in process, until we are sure problems not in views too lock)
+SQLAlchemy==1.2.19
```
Comment (jpmckinney) [inline requirements.in]: I prefer `SQLAlchemy<1.3` which is what we actually want. If security updates are released for 1.2, we're happy to have them.

**Comment by odscjames** [comment]
Deploy commit merged and deployed. This branch Merged and deployed.

**Pull request merged**